utils/ansible.py

```diff
@@ -0,0 +1,281 @@
+from distutils.sysconfig import get_python_lib
+import tempfile
+from os import listdir, mkdir, path
+from shutil import copy, copyfile, rmtree
+from subprocess import check_output, CalledProcessError, STDOUT
+
+from fauxfactory import gen_alphanumeric
+from utils import conf
+from utils.providers import get_crud
+
+from git import Repo
+from yaml import load, dump
+
+
+yml_path = path.join(path.dirname(__file__) + "/manageiq_ansible_module/")
+library_path = get_python_lib()
+yml_templates_path = path.dirname(__file__) + "/ansible_conf/"
+providers_basic_script = "providers_basic_script.yml"
+users_basic_script = "users_basic_script.yml"
+basic_script = "basic_script.yml"
+basic_script_bk = "basic_script_bk.yml"
+yml = ".yml"
+random_token = str(gen_alphanumeric(906))
+random_miq_user = str(gen_alphanumeric(8))
+local_git_repo = "manageiq_ansible_module"
+pulled_repo_library_path = local_git_repo + "/library"
+remote_git_repo_url = "git://github.com/dkorn/manageiq-ansible-module.git"
+
+
+def create_tmp_directory():
+    global lib_path
```

Comment:
Can you elaborate please? I'm not sure I modify the variable anywhere. I do need this variable later on to remove the library path. 
    rmtree(lib_path, ignore_errors=True)